vagrant/concourse_post_deploy.sh

```diff
@@ -1,6 +1,9 @@
 #!/bin/bash
 gardenDir="/var/vcap/data/garden"
 
+echo "Removing packer ssh key..."
+sed "/packer /d" -i /home/ubuntu/.ssh/authorized_keys
```

Comment:
Could you change this to the following, which will delete everything except the `vagrant` key:
```
sed -ie '/ vagrant$/ ! { d }' .ssh/authorized_keys
```
So that it's a bit more resilient to:
- the comment in the SSH pubkey changing in the Concourse AMI build process
- the directory which comes from `override.ssh.username` changing in the `Vagrantfile`